force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Apex extra: Test without data factory or builder
#### Severity
Error

#### Finding
The variable `Contact()` is created directly within a test method. Please use a data builder or a data factory to centralise the creation of test data and limit the impact of any changes to validation rules.

#### Why is this a problem?
Test methods should not create record instances directly as they would break whenever adding validation rules. As a best practice, the creation of test data records should be centralised using test data factories or builders.

#### Help and documentation

* [Create Test Data for Apex Tests (Trailhead)](https://trailhead.salesforce.com/en/modules/apex\_testing/units/apex\_testing\_data) 
* [Use of Obsolete Function (CWE)](https://cwe.mitre.org/data/definitions/477.html)